src/Features/Lsif/Generator/Program.cs

```diff
@@ -42,8 +43,18 @@ public static Task Main(string[] args)
         private static async Task GenerateAsync(FileInfo? solution, FileInfo? project, FileInfo? compilerInvocation, FileInfo? binLog, string? output, LsifFormat outputFormat, string? log)
         {
             // If we have an output file, we'll write to that, else we'll use Console.Out
-            using var outputFile = output != null ? new StreamWriter(output) : null;
-            var outputWriter = outputFile ?? Console.Out;
+            using var outputFile = output != null ? new StreamWriter(output, append: false, Encoding.UTF8) : null;
+            TextWriter? outputWriter;
```

Comment:
```suggestion
            TextWriter outputWriter;
```

Reply:
oh wow, I didn't even know that was allowed